server/app/featureflags/FeatureFlags.java

```diff
@@ -21,6 +21,7 @@ public final class FeatureFlags {
   private static final String FEATURE_FLAG_OVERRIDES_ENABLED = "feature_flag_overrides_enabled";
   public static final String APPLICATION_STATUS_TRACKING_ENABLED =
       "application_status_tracking_enabled";
+  public static final String PROGRAM_READ_ONLY_VIEW_ENABLED = "program_read_only_view_enabled";
```

Comment:
Can you please alphabetize these 3 while you're here (excluding the first one: FEATURE_FLAG_OVERRIDES).